targets/export-elasticsearch/src/services/export.ts

```diff
@@ -49,9 +49,27 @@ export class ExportService {
       try {
         if (!process.env.DISABLE_INGESTER) {
           if (environment === Environment.preproduction) {
+            await sendMattermostMessage(
+              `La mise à jour de la préproduction a été lancée par ${exportEs.user?.name}. 😎`,
+              process.env.MATTERMOST_CHANNEL_EXPORT
+            );
             await runWorkerIngesterPreproduction();
+            await sendMattermostMessage(
+              "La mise à jour de la préproduction s'est terminée. 😁",
+              process.env.MATTERMOST_CHANNEL_EXPORT
+            );
           } else {
+            await sendMattermostMessage(
+              "La mise à jour de la production a été lancée. 🚀",
```

Comment:
on peut mettre le user ici aussi
```suggestion
              "La mise à jour de la production a été lancée par ${exportEs.user?.name}. 🚀",
```